zipline/data/data_portal.py

```diff
@@ -637,7 +638,51 @@ def _check_extra_sources(self, asset, column, dt):
 
                 raise KeyError
 
-    def get_spot_value(self, asset, field, dt, data_frequency):
+    def _get_overnight_adjustments(self, assets, field, dt):
+
+        if not isinstance(assets, (list, tuple)):
```

Comment:
As far as I can tell this is only ever called once, by our own internal code.  Why are we doing a type coercion here?